shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -193,14 +193,14 @@ static bool ClipRRectContainsPlatformViewBoundingRect(const SkRRect& clip_rrect,
 void FlutterPlatformViewsController::OnCreate(FlutterMethodCall* call, FlutterResult& result) {
   NSDictionary<NSString*, id>* args = [call arguments];
 
-  long viewId = [args[@"id"] longValue];
+  int64_t viewId = [args[@"id"] longLongValue];
```

Comment:
Any reason not to use `long long` instead?
```suggestion
  long long viewId = [args[@"id"] longLongValue];
```

Reply:
Using int64_t was recommended in the google c++ style guide for the below reason:
https://google.github.io/styleguide/cppguide.html#Integer_Types
```
The standard library header <cstdint> defines types like int16_t, uint32_t, int64_t, etc. You should always use those in preference to short, unsigned long long and the like, when you need a guarantee on the size of an integer.
```

Reply:
That style guide makes sense, but I think it's saying that the value coming out of `args[@"id"]` should be enforced to be a `int64_t` since that's what it's set to on the dart side.  But at the point `-longLongValue` is being used it's a `long long`.  This is all pedantic though since `int64_t` is typedef'd to `long long` on macOS, watchOS, tvOS, and iOS.